### PR TITLE
Take the user name from the JWT and not the CSR database.

### PR DESCRIPTION
**Pull request by nickmcmahon01**

### CODE REVIEW

File: integration-tests/mockApis/auth.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const { stubFor, getRequests } = require('./wiremock')
 const createToken = () => {
   const payload = {
     sub: 'ITAG_USER',
-    user_name: 'ITAG_USER_USE_SUB',
+    name: 'SARAH ITAG',
```
Comment (astroash) [inline integration-tests/mockApis/auth.js:7]: You've changed the name of the key here, was this originally an error with the test data or are we renaming it?

Comment (nickmcmahon01) [reply integration-tests/mockApis/auth.js:7]: the Auth Team have dropped user_name from the JWT in favour of Sub and I've asked them to add name. two unrelated things but that's what a JWT looks like now.

File: server/routes/login.js [hunk 2]
```diff
@@ -40,11 +40,7 @@ module.exports = () => (router) => {
...
       if (inputTwoFactorCode === userAuthenticationDetails[0].TwoFactorAuthenticationHash) {
-        req.user.employeeName = await getStaffMemberEmployeeName(
-          userAuthenticationDetails[0].ApiUrl,
-          utilities.getStartMonth(),
-          req.user.token,
-        )
+        req.user.employeeName = jwtDecode(req.user.token).name
 
         await userAuthenticationService.updateUserSessionExpiryAndLastLoginDateTime(
           req.user.username,
```
Comment (astroash) [inline server/routes/login.js:46]: I know this is existing code, but is there a reason we are adding the value to the request object, then passing it to a function here? I may be missing something but it seems unusal to mutate the request object, unless we specifically need it for some pre response middleware. If not declaring the name as a variable would be more readable here.

Comment (nickmcmahon01) [reply server/routes/login.js:46]: I'm not sure why we're mutating the request object but req.user.employeeName and req.user.username are different. "employeeName" is like "Nick McMahon" and "username" is my Quantum Id

Comment (astroash) [reply server/routes/login.js:46]: I've been reading through the codebase to see if it's used anywhere, and I can't seem to find it, but it's hard to say without any preexisting tests. Happy to take your call on whether this should be changed or moved elsewhere.

**Pull request merged**